packages/replies/src/types.ts

```diff
@@ -57,3 +57,8 @@ export interface ReplyBoard {
   childMessagesCount: number
   pinnedMessages: Reply[]
 }
+
+export interface Placeholders {
+  replyPlaceholder: string
+  childReplyPlaceholder: string
```

Comment:
하나만 설정할 수도 있는 거니까(로직도 그러하니까) 각각 Optional이어야 할 것 같고, placeholder라는 게 반복되니까 줄여줄 수 있을 것 같아요

```suggestion
  reply?: string
  childReply?: string
```

Reply:
`reply`란 단어랑 `message`란 단어가 자꾸 섞이고 있는 상태인데 일단 Replies 컴포넌트의 Props에선 reply를 쓰고 있으므로.. 이렇게 해봤어요.